packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -790,6 +790,20 @@ const ArrayMixin = Mixin.create(Enumerable, {
 
     This method works much like the more generic `find()` method.
 
+    Usage Example:
+    ```javascript
+    let users = [
+      { id: 1, name: 'Yehuda', isTom: false },
+      { id: 2, name: 'Tom', isTom: true },
+      { id: 3, name: 'Melanie', isTom: false },
+      { id: 4, name: 'Leah', isTom: false }
+    ];
+
+    users.findBy('id', 4); // Leah
```

Comment:
The value returned is the full object, right? Can you update the comment bit to reflect that so readers aren't accidentally confused?

Reply:
Ah, good catch! I can totally see how that could be confusing.